package.json

```
@@ -1,13 +1,20 @@
 {
   "name": "pejuangkode.com",
-  "version": "0.0.1",
+  "version": "1.0.0",
+  "description": "Website pejuangkode.com",
+  "main": "N/A",
+  "repository": "https://github.com/pejuang-kode/pejuangkode.com",
+  "bugs": "https://github.com/pejuang-kode/pejuangkode.com/issues",
+  "license": "MIT",
+  "private": true,
   "scripts": {
     "dev": "next",
     "build": "next build",
     "start": "next start"
   },
   "dependencies": {
     "next": "latest",
+    "next-seo": "^4.17.0",
```

Tolong dependency-nya dikunci ke versi ini dengan cara menghilangkan karakter _caret_ `^` ini supaya tidak ada perubahan yang tidak terencara Mas.
```suggestion
    "next-seo": "4.17.0",
```

---

Setuju, untuk menghindari error antar dependecy 👌